### DISCUSSION TITLE
是否可以支持增加可以控制全局的一些配置项

### DISCUSSION THREAD

**Discussion by ZZFFLL** [lobehub/lobehub#2290] [General Question]
对于分享给其他人使用的场景：
1、在共享使用时，需要对全局的上下文做限制（包括但不限于默认的“随便聊聊”会话）：例如自动创建话题、限制历史消息数、是否开启历史消息长度压缩阈值、开启单次回复限制等这些配置项通过环境变量等方式来强制指定而不让用户自行修改
2、设置这些的前提是管理员来决定，管理员决定哪些配置项是用户可以自行修改的

**Comment by arvinxx** [comment]
@ZZFFLL 现在有 Feature Flags , #1894 ，但你说的可能有点过于复杂了，我不确定基于 FF 能否支持上。另外我应该不会主动做这个需求，如果有需要欢迎 PR 共建~

**Reply by ZZFFLL** [reply]
确实会复杂，相当于得加入一套功能权限控制

**Reply by imwito** [reply]
请问Feature Flags是类似 #1430 描述的吗，这样比导出导入方便很多。
